wgsl/index.bs

```diff
@@ -2893,7 +2893,7 @@ The size may include non-addressable padding at the end of the type.
 Consequently, loads and stores of a value might access fewer memory locations
 than the value's size.
 
-Alignment and size of types are defined recursively in the
+Alignment and size of non-opaque types are defined recursively in the
```

Comment:
Is bool opaque or not?
Host-shareable does not include bool.

The table today has host-shareable in the heading.  

I suggest reverting back to the baseline text here.

Reply:
What about override sized wg storage? We're supposed to use size there for workgroup memory limits.